doofinder.php

```diff
@@ -53,7 +53,7 @@ public function __construct()
     {
         $this->name = 'doofinder';
         $this->tab = 'search_filter';
-        $this->version = '4.0.1';
+        $this->version = '4.0.2';
```

Comment:
I think this should be:

```suggestion
        $this->version = self::VERSION;
```

You should change the constant instead, which is used in other places like in `config.php`.